spec/features/catalog_view_spec.rb

```diff
@@ -201,14 +201,31 @@
 
     it "renders a view content link" do
       url = "https://webspace.princeton.edu/users/mudd/Digitization/MC148/MC148_c07608.pdf"
-      expect(page).to have_selector("a[href=\"#{url}\"]", text: "View Content")
+      expect(page).to have_selector("a[href=\"#{url}\"]", text: "View digital content")
```

Comment:
Standards @sdellis looked at during the hyrax sprint suggested title case for buttons. It would be good to have this come out as "View Digital Content"